### ISSUE TITLE
Incorrect Closing Bracket Indentation on Emacs 28.1

### ISSUE THREAD

**Issue by Stumble** [redguardtoo/emacs.d#975]
Emacs 28.1 installed through snap.
$ emacs --version 
GNU Emacs 28.1

```
if (cond) {
    for (auto a : v) {}
                      ^ when press enter in between brackets
}

expected:
if (cond) {
    for (auto a : v) {
        // cursor is here
    } 
}

but on Emacs 28.1:
if (cond) {
    for (auto a : v) {
        // cursor is here
}       // <-- incorrect indentation.
}

```
Same issue as this: https://github.com/hlissner/doom-emacs/issues/5200

**Comment by redguardtoo** [comment]
because 28.1 enabled `electirc-pair-mode`? Raise a issue to Emacs core team or the author of that mode.

**Comment by phye** [comment]
Same issue here. Really annoying.

**Comment by phye** [comment]
Someone already raised an issue, but still no updates from upstream...
https://debbugs.gnu.org/cgi/bugreport.cgi?bug=55340

**Comment by sbwcwso** [comment]
emacs 28.2 still have this problem, any solutions now?

**Comment by redguardtoo** [comment]
fixed c773ce13

**Comment by Stumble** [comment]
> fixed [c773ce1](https://github.com/redguardtoo/emacs.d/commit/c773ce13259da5922520ac679eb75557e9afb199)

Thanks! Just upgraded to 28.2 and verified!
